### PR TITLE
Add Ruby 2.7.2 and update Ruby versions in travis.yml

### PR DESCRIPTION
**Pull request by yucao24hours**
Added Ruby 2.7.0 to build matrix.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ rvm:
 - 2.4.9
 - 2.5.7
 - 2.6.5
+- 2.7.0
```
Comment (koic) [inline .travis.yml:8]: Can you update latest Ruby versions?

- 2.4.10
- 2.5.8
- 2.6.6
- 2.7.2

Comment (yucao24hours) [reply .travis.yml:8]: Sure, and I'll change this PR title to `Add Ruby 2.7.2 and update Ruby versions in travis.yml` .

Comment (koic) [reply .travis.yml:8]: It seems that the update from 2.7.0 to 2.7.2 has been forgotten :-)

Comment (yucao24hours) [reply .travis.yml]: Oh sorry... I've added it! 😇

Comment (koic) [reply .travis.yml]: CI is failing.
https://travis-ci.org/github/esminc/adhoq/jobs/733932214#L243-L248

Can you update from Ruby 2.4.9 to 2.4.10 for exclude matrix as well?
https://github.com/esminc/adhoq/blob/71f9cec0520151f883444a1c0deac5435743a68d/.travis.yml#L32-L35

**Comment by koic** [comment]
This looks good to me. Can you squash your commits into one?

**Comment by yucao24hours** [comment]
Thank you for reviewing! I've pushed the squashed commit.

**Comment by koic** [comment]
Great thanks!

**Pull request merged**